jenkins_pipelines/performance_testsuite/Jenkinsfile_performance_testsuite

```diff
@@ -55,6 +56,13 @@ pipeline {
                 sh "ssh $ssh_flags -tt root@$locust_node $exec_locust -c 400"
             }
         }
+
+        stage('Apply patches in evil-minions') {
+            steps {
+                sh "chmod +X prod/performance/sumaform/$apply_patches_script"
```

Comment:
i would do this directly in sumaform, namely commit a chmod +x script in sumaform